spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -649,6 +649,9 @@ PaymentProcessor pp = anyPaymentProcessor.select(qualifier).get().process(paymen
 
 In this example, the returned bean has qualifier `@Synchronous` or `@Asynchronous` depending upon the value of `synchronously`.
 
+Qualifiers specified at `Instance` injection point or via via the `Instance.select()` method are used to perform typesafe resolution.
```

Comment:
Maybe I would use "specified by" but I would unify this in this PR - in the next file there is "specified in". WDYT?